common/app/views/fragments/atoms/snippets/snippetImage.scala.html

```diff
@@ -0,0 +1,25 @@
+@import layout.ContentWidths.MainMedia
```

Comment:
I created a simpler alternative as it was adding the `u-responsive-ratio` div, though looking again it appears I can avoid that with `isImmersiveMainMedia = true`